source/common/common/utility.cc

```diff
@@ -102,6 +102,30 @@ std::string StringUtil::subspan(const std::string& source, size_t start, size_t
   return source.substr(start, end - start);
 }
 
+std::vector<std::string> StringUtil::tokenize(const std::string& source,
```

Comment:
OK, fair enough, can you add a few more comments to utility.h that show the difference between split and tokenize? It wasn't clear to me.